stack2struct.go

```diff
@@ -58,12 +60,19 @@ func Parse(trace []byte, stack stackTrace) {
 	}
 }
 
+// LoadGoErrorStack loads the strack trace (given as frames) into the given stack.
+func LoadGoErrorStack(frames []goerrors.StackFrame, stack stackTrace) {
+	for _, frame := range frames {
+		stack.AddEntry(frame.LineNumber, frame.Package, frame.File, frame.Name)
+	}
+}
+
 // extractPageName receives a trace line and extracts packageName and
 // methodName.
 func extractPackageName(line string) (packageName, methodName string) {
 	packagePath, packageNameAndFunction := splitAtLastSlash(line)
 	parts := strings.Split(packageNameAndFunction, ".")
-	
+
```

Comment:
Can you revert these whitespaces adjusts to making this PR cleaner?

I mean, an additional PR could be written to handle code styling. But, would be nice this one to have only code related to the issue it aims to fix.